tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -26,6 +27,8 @@ public class Tracer : ITracer, IDatadogTracer, IDatadogOpenTracingTracer
     {
         private static readonly object GlobalInstanceLock = new();
 
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<Tracer>();
```

Comment:
I don't think this is used?
```suggestion
        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<Tracer>();
```